code/game/objects/items/tanks/tanks.dm

```diff
@@ -23,6 +23,10 @@
 	var/volume = 70
 	/// Mob that is currently breathing from the tank.
 	var/mob/living/carbon/breathing_mob = null
+	/// Attached igniter assembly, used to ignite its contents
+	var/obj/item/assembly_holder/tank_assembly = null
+	/// Whether or not it can explode when it receives a signal
+	var/status = FALSE
```

Comment:
This needs a more descriptive var name for something meant to represent something so destructive.